ultravox/utils/string_helpers.py

```diff
@@ -0,0 +1,63 @@
+import re
+import sys
+import unicodedata
+
+
+def fix_hyphens(arg: str):
+    return re.sub(r"^--([^=]+)", lambda m: "--" + m.group(1).replace("-", "_"), arg)
+
+
+def sanitize_name(name: str) -> str:
+    """
+    Sanitize a string to be safely used as a valid filename, folder name, or Python module name.
+
+    Parameters:
+        name (str): The input string to sanitize.
+
+    Returns:
+        str: The sanitized name.
```

Comment:
I don't follow the logic of this file at all. Did you write this? If not, what is the source?